src/workerd/io/tracer.c++

```diff
@@ -334,7 +339,11 @@ void WorkerTracer::setEventInfo(
       .entrypoint = mapCopyString(trace->entrypoint),
     };
 
-    writer->report(context, tracing::Onset(cloneEventInfo(info), kj::mv(workerInfo), kj::none));
+    // TODO(o11y): At this time, the onset spanId is always zero. We could use the invocation span
```

Comment:
Seems to me like the tracing context should have the entire invocation's context and part of that might be the parent span id for the invocation
Then we could stub 0 there now while still providing ourselves with one clear place to fix when we sort out context propagation

Reply:
FWIW, I was of the impression that that's how the code is already architected...